tutorial/versioning/lockfiles.rst

```diff
@@ -315,9 +315,12 @@ configuration, and then merge them:
 This multiple-clean + merge operation is not something that developers should do, only CI
 scripts, and for some advanced CI flows that will be explained later.
 
+.. note:
+
+   It is possible to lock down to package revisions, but this would be not recommended for
+   most use cases, and should only be used in extreme and problematic cases.
+
```

Comment:
```suggestion
```

We have even removed this from the command help and from the docs, it is not mentioned anywhere.  So it is not really possible at the moment.